packages/components/src/components/accordion/_accordion.scss

```diff
@@ -98,12 +98,17 @@
     // Transition property for when the accordion closes
     transition: padding motion(standard, productive) $duration--fast-02;
     padding-left: $carbon--spacing-05;
-    padding-right: 25%;
+    padding-right: $carbon--spacing-05;
 
-    @include carbon--breakpoint-down('md') {
+    // Custom breakpoints based on issue #4993
+    @media (min-width: 480px) {
```

Comment:
I think you could use `carbon--breakdown-up` if you want! e.g.

```scss
@include carbon--breakpoint-up(480px) {
  // ...
}
```

But it's basically the same thing 😂

Reply:
Oh nice, I wasn't sure if we had one for constructing custom breakpoints. I'll update it for consistency.